cms/templates/js/course-video-settings-update-settings-footer.underscore

```diff
@@ -0,0 +1,13 @@
+<button class='button-link action-cancel-course-video-settings' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
```

Comment:
@srpearce We are still not clear on this one 😃  

For the context:
This button lets user to discard changes user have made currently.

Previous strings looked at:
- Cancel (this was rejected because this becomes a bit confusing with Close button)
- Reset
- Restore Defaults (Defaults here would imply None settings values. and user may think there are some defaults already)

**The thing we are trying is for user to let go of the changes he has not yet saved**. He may wish to discard the settings he is doing on UI (saved settings can be the ones before we opened the page or user could have saved in current operations on the current settings pane)

Reply:
I like "Discard Changes", in that case. Does that work?